### PR TITLE
Fix install option for pillow

### PR DESCRIPTION
**Pull request by unnonouno**
I appended global options to install pillow.

### CODE REVIEW

**Comment by gwtnb** [comment]
LGTM

File: docker.py [hunk 1]
```diff
@@ -312,7 +312,11 @@ def set_env(env, value):
 
 
 def run_pip(requires):
-    return 'RUN pip install -U "%s"\n' % requires
+    if 'pillow' in requires:
+        return ('RUN pip install --global-option="build_ext" '
+                '--global_option="disable-jpeg" -U "%s"\n' % requires)
```
Comment (gwtnb) [inline docker.py]: `--global-option` (not `--global_option`)

Comment (unnonouno) [inline docker.py]: Thanks. I fixed it.

File: docker.py [hunk 2]
```diff
@@ -312,7 +312,11 @@ def set_env(env, value):
 
 
 def run_pip(requires):
-    return 'RUN pip install -U "%s"\n' % requires
+    if 'pillow' in requires:
+        return ('RUN pip install --global-option="build_ext" '
+                '--global-option="disable-jpeg" -U "%s"\n' % requires)
```
Comment (gwtnb) [inline docker.py]: `--disable-jpeg`

**Comment by gwtnb** [comment]
LGTM

**Pull request merged**